source/_components/homekit.markdown

```diff
@@ -294,6 +294,7 @@ The following components are currently supported:
 | cover | WindowCovering | All covers that support `open_cover`, `stop_cover` and `close_cover` through value mapping. (`open` -> `>70`; `close` -> `<30`; `stop` -> every value in between) |
 | device_tracker | Sensor | Support for `occupancy` device class. |
 | fan | Fan | Support for `on / off`, `direction` and `oscillating`. |
+| fan | Fan | All fans that support `speed` and `speed_list` through value mapping. (first entry of `speed_list` -> `0`; last entry of `speed_list` -> `100`; entry in between -> patching percentage) |
```

Comment:
`matching percentage`? Can you add a line, that the values need to be in ascending order?